### ISSUE TITLE
Question regarding tabular figures

### ISSUE THREAD

**Issue by gorjious** [googlefonts/googlesans#74]
In the Text, numerals and currency symbols are at Cap Height, and in the Display, they are lower.
Activating Tabular Figures in the Display causes the numerals to go up to the Cap Height, while the currency symbols remain at the lower height.

With Tabular Figures activated, should the numerals be at the same lower height as the currency symbols? 
If there is a need to have numerals and currency symbols available at both heights in the Display, should the Case feature cover that (this would require additional glyphs i.e. zero.cap.tb)?

[GS_issue_TabularFigures.pdf](https://github.com/googlefonts/googlesans/files/5037102/GS_issue_TabularFigures.pdf)

<img width="1392" alt="Screenshot 2020-08-06 at 12 11 54" src="https://user-images.githubusercontent.com/23149420/89567592-29f7b580-d7df-11ea-84b4-03392f1fa92a.png">

**Comment by chrissimpkins** [comment]
My opinion is that it would be nice to push everything to cap height across the board.  We are running into issues with the alignment of the tops of the figures in Linux environments.  The FT autohinter doesn't like the sub cap height position and we are getting different pixel shifts for the round top + overshoot and flat top numerals.  

Thoughts about this issue Edd? @Colophon-Foundry

**Comment by chrissimpkins** [comment]
Ping @Colophon-Foundry.  We are also working on these designs in the Hebrew work that is being merged now.

**Comment by chrissimpkins** [comment]
Related https://github.com/googlefonts/googlesans/issues/31

**Comment by Colophon-Foundry** [comment]
This is a good point. I would suggest one of three options:

**Option 1**
1. Change tabular figures to be non-cap height. This will bring consistency to the font by activating tabular figures.
2. Add in .tf.cap which creates cap-height tabular form characters of both numbers and currencies

**Option 2**
1. Change tabular currency forms to be cap height

**Option 3**
1. Change all forms to cap-height


I would say Option 1 makes the most sense, but it does bring with it a weight of duplication of character count. Option 2 makes the issue amplified (IMHO). Option 3 would be the best solution, but it changes the design space quite a lot (but does shave off a lot of characters as we could dump all of the `cap` forms!)

*Note all of these comments relate to the Sans Max Opsize only.

**Comment by chrissimpkins** [comment]
IMO if we are going to stick with the below cap height figure forms (includes default and all alternate designs), we need to address #31.  I consider this a bug and have seen this myself in Google products on Linux.  Numerals don't look good on that platform and they are everywhere in Google products.  Marc Foley suggested that perhaps we nudge the non-rounded forms up slightly so that the autohinter pixel shifts function like they do with the rounded top forms.  We would need to experiment with how much is necessary while still keeping them below cap height.

@danielgrumer You worked on these designs for the Hebrew sheqel currency symbol.  Do you have any thoughts about height/consistency?

**Comment by Colophon-Foundry** [comment]
I'd be in favour of moving all numbers to the cap height forms (we could keep the current lower forms in a stylistic set?) but I guess this would affect a lot of designs/products etc!

**Comment by gorjious** [comment]
Are the issues in Linux occurring only at small sizes? Could those cases be a matter where GS min opsz (with cap height numerals/currency symbols) is a better choice to use?

**Comment by chrissimpkins** [comment]
> Are the issues in Linux occurring only at small sizes? Could those cases be a matter where GS min opsz (with cap height numerals/currency symbols) is a better choice to use?

See the images in #31.  The first image is where it was identified in a production Google product (note the problem is not shown there, this was my SS on macOS to show where the problem was found, it only occurs on Linux, not Win/macOS).  I don't believe that the use case fell in the GS Text size range.  The other images were generated with a testing site that I made.  It happens at large text sizes too.  We attempted to work around it with manual hinting and then recognized that this is due to the FreeType autohinter on GNU/Linux.  This is an issue that we can't work around with font hinting.  Marc's suggestion was to see if we can fool the FT autohinter by pushing the tops of the flat top forms to a level that forces it to snap one pixel higher rather than one pixel lower.  This might achieve alignment and still preserve the sub-cap height metrics if this is preferred for aesthetic reasons in max opsz designs.
